src/navigation/wallet/WalletStack.tsx

```diff
@@ -342,6 +347,13 @@ const WalletStack = () => {
           name={WalletScreens.GLOBAL_SELECT}
           component={GlobalSelect}
         />
+        <Wallet.Screen
```

Comment:
question: how to test this?

Reply:
you need to have more than 2 backup keys and use the general scanner for the invitation code